client/src/app/domain/models/poll/generic-poll.ts

```diff
@@ -20,6 +21,7 @@ export interface PollData {
     options: OptionData[];
     options_as_observable: Observable<OptionData[]>;
     global_option: OptionData;
+    content_object?: BaseViewModel<any>;
```

Comment:
Is it possible to be more specific about the type here?

Reply:
Not completely sure but I think this could be
```suggestion
    content_object?: BaseViewModel<ViewUser | ViewPollCandidateList>;
```

Reply:
I agree that I may want to be a bit more specific here, but in this place it would be `content_object?: ViewMotion | ViewAssignment | ViewTopic`, and then in ln. 47 it should be `content_object?: ViewMotion | ViewUser | ViewPollCandidateList`
I'll look into it